docs/IR.md

```diff
@@ -351,6 +353,18 @@ There are two ways to leave an optional input or output unspecified: the first,
 
 Each node referring to an operator with optional outputs MUST provide a name for each output that is computed and MUST NOT provide names for outputs that are not computed.
 
+Optional inputs and outputs of the above kind are referred to as _static-optional_.
+
+##### IR-8 Version and Later
```

Comment:
I like the sub sections here, which are very clear for different IR versions.

nit: I feel `#####` is a little small? (Even smaller then the content below?) Will simply using bold text better?
```suggestion
**IR-8 Version and Later**
```

Reply:
Done, thanks!